src/Mailer/Transport/MailTransport.php

```diff
@@ -41,7 +41,7 @@ public function send(Message $message): array
         $to = $message->getHeaders(['to'])['To'];
         $to = str_replace("\r\n", '', $to);
 
-        $eol = $this->getConfig('eol', PHP_EOL);
+        $eol = $this->getConfig('eol', "\r\n");
```

Comment:
The php bug listed above was fixed in php8. https://bugs.php.net/bug.php?id=47983

Reply:
I'm hoping that MTAs are ok with CRLF in headers. The other option would be to change the default based on the PHP version.

Reply:
It's the standard and if php 8 is doing it then we're as compatible as php is.

Reply:
I wonder if php 7.4 will throw errors because PHP uses `\n` while cakephp uses `\r\n`. We might need version-specific default?

Would be nice to test that at least.

Reply:
Yeah, using `\n` for PHP7 would make this a safer change.